ESP_ATMod/ESP_ATMod.ino

```diff
@@ -134,6 +138,8 @@ uint16_t gsCipSslSize = 16384;		// command AT+CIPSSLSIZE
 bool gsSTNPEnabled = true;			// command AT+CIPSNTPCFG
 int8_t gsSTNPTimezone = 0;			// command AT+CIPSNTPCFG
 String gsSNTPServer[3];				// command AT+CIPSNTPCFG
+uint8_t gsServersMaxConn = 5;			// command AT+CIPSERVERMAXCONN
+uint32_t gsServerConnTimeout = 180;	// command AT+CIPSSTO

```

Comment:
The timeout value is given in milliseconds, therefore this value is effectively 0. I think it should be 180*1000.

Reply:
according to AT commnads reference the CIPSTO  value is in seconds

Reply:
Yes, it is, but the value in gsServerConnTimeout is in milliseconds, AFAIK